lib/api/bucketPutACL.js

```diff
@@ -100,19 +101,22 @@ export default function bucketPutACL(authInfo, request, log, callback) {
         aclUtils.parseGrant(request.headers
             ['x-amz-grant-full-control'], 'FULL_CONTROL');
 
+    let corsHeaders;
```

Comment:
Can you get this value at the end of the waterfall, rather than using the outer scope?